src/pymail

```diff
@@ -25,56 +26,74 @@ install_path = "/home/martijn/Coding/git/PyMail/src/"
 # Functions
 
 def send_email_from_terminal(use_sys_argv):
-    """Send an email"""
-    if use_sys_argv == False:
-        to_addr = raw_input("Receiver address = ")
-        subject = raw_input("Subject = ")
-        body = raw_input("Body = ")
-    elif use_sys_argv == True:
-        to_addr = sys.argv[2]
-        subject = sys.argv[3]
-        body = sys.argv[4]
+    """Set the mail-components from sysargv or from user input
+    depending on if use_sys_argv is True or False. Send mail after
+    setting variables.
+    """
+
+    if use_sys_argv:
+    	# sysargv 2, 3, and 4
+        to_addr, subject, body = [sys.argv[x] for x in range(2,5)]
+    else:
+        to_addr, subject, body = [raw_input(x + " = ") for x in ['Receiver address', 'Subject', 'Body']]
 
     send_mail(to_addr, subject, body, True)
 
-def send_email_from_file(delete_concept):
-    """Sends an email read from a file, given in the commandline"""
-    concept_file_obj = open(sys.argv[2], 'r')
-    concept_file = concept_file_obj.read()
-    split_data = concept_file.split('--- --- ---')
 
-    to_addr = split_data[0].replace('\n','')
-    subject = split_data[1].replace('\n','')
-    body = split_data[2]
+def send_email_from_file(delete=False):
+    """Gets the email components from a given file
+    and sends an email with said components.
+    Will also delete the file if the delete variable == True
+    """
+
+    to_addr, subject, body = None
+
+    # This will assure that the file is closed
+    with open(sys.argv[2], 'r') as file:
+    	to_addr, subject, body = file.read().split('--- --- ---')[:3]
```

Comment:
Added the `[:3]` since you only wanted the three first items, and the user might fuck up the file